consensus/src/util/mod.rs

```diff
@@ -7,3 +7,20 @@ pub mod mock_time_service;
 pub mod time_service;
 #[cfg(test)]
 mod time_service_test;
+
+/// Helper function to record metrics for external calls.
+/// Include call counts, time, and whether it's inside or not (1 or 0).
+/// It assumes a OpMetrics defined as OP_COUNTERS in crate::counters;
+#[macro_export]
+macro_rules! monitor {
```

Comment:
This can be even part of monitoring module.
OP_COUNTERS can still be in consensus, it will work if you replace `crate` with `$crate`,

Reply:
Actually, you don't even need to replace, `crate:` is the right word here

Reply:
yeah I just tried, $crate actually refers to where the macro is defined